playground/alias/vite.config.js

```diff
@@ -1,9 +1,8 @@
-const path = require('node:path')
+// @ts-check
```

Comment:
I think it would be nice to adjust the tsconfig so this comment isn't needed everywhere 🤔

Reply:
Yup that include glob looks good to me 👍